templates/index.html

```diff
@@ -0,0 +1,112 @@
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="../css/style.css" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <p>Hello World</p>
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+    <div class=nav>
+
+        <ul>
+          <li class="listitems">
+            <a href="#">About</a>
+          </li>
+          <li class="listitems">
+            <a href="#">work</a>
+          </li>
+          <li class="listitems">
+            <a href="#">contact</a>
+          </li>
+        </ul>
+  
+    </div>
+    <!-- start sectionone -->
+    <section class="sectionone pd-y" id="sectionone">
+      <div class="sectionone__wrapper">
+        <h1>Hey I am Mimic</h1>
+        <h3 class="center">a web developer</h3>
+      </div>
+    </section>
+    <!-- end sectionone -->
+    <section>
+      <div class="flexheader pd-y" id="flexheader">
+        <p class="center">these are some of my projects</p>
+
+        <div class="flex">
+          <a href="#">
+            <img
+              src="https://icatcare.org/app/uploads/2018/07/Thinking-of-getting-a-cat.png"
+          /></a>
+          <a href="#">
+            <img
+              src="https://www.rd.com/wp-content/uploads/2021/01/GettyImages-1175550351.jpg"
+          /></a>
+          <a href="#">
+            <img
+              src="https://cdn.britannica.com/39/7139-050-A88818BB/Himalayan-chocolate-point.jpg"
+          /></a>
+          <a href="#">
+            <img
+              src="https://www.boredpanda.com/blog/wp-content/uploads/2022/06/funny-pics-cats-with-powerful-auras-cover_800.png"
+              alt="this is image four"
+          /></a>
+          <a href="#">
+            <img
+              src="https://mediaproxy.salon.com/width/1200/height/675/https://media.salon.com/2022/05/cats-party-0516221.jpg"
+          /></a>
+          <a href="#">
+            <img
+              src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRzklJZyjjFXAqKOZ3qhEObLEXPTxRrEc16Dw&usqp=CAU"
+          /></a>
+          <a href="#">
+            <img
+              src="https://moderncat.com/wp-content/uploads/2014/03/tortoiseshell-kitten-closeup-p-46771525-940x640.jpg"
+          /></a>
+        </div>
+      </div>
+    </section>
+    <!-- start sectiontwo -->
+    <section class="sectiontwo pd-y" id="sectiontwo">
+      <div class="sectiontwo__wrapper">
+        <h1 class="center">Let's work together...</h1>
+        <p class="center">How do you take your coffee?</p>
+        <div>
```

Comment:
- [ ] I think you forgot to put (ul) tag instead of (div) tag, so don't worry it happens to us all we are humans ☺️ ,so after some time, you will exceed this problem 😎